scripts/travis/build-windows.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
is this script necessary? I.e. why not just call `make build-windows` from .travis?

Reply:
Yup, I can move it to just running `make build-windows` I was following convention and created a script file. I'll change that up.

Reply:
Let's be nice to the next person that will touch this feature and keep it consistent with the other parts of the build :)

Reply:
I'm confused by the two comments - should I use `make build-windows` or call the script?

Reply:
I would say that the script is desirable, for consistency's sake. But @yurishkuro has the final word on this.

Reply:
Personally I prefer minimalist approach, of the script doesn't add anything then why have it? 

But since it's here already, it's fine to leave it.